steps/keras/models.py

```diff
@@ -3,8 +3,10 @@
 import numpy as np
```

Comment:
@jakubczakon I think that Embeddings should go to the new file: `embeddings.py`. These are not models, but embeddings produces by the model. I think it would be clearer to put them in the: steps -> embeddings.py.
Firstly, embeddings are useful in multiple places, so it should be in `steps`. Secondly, we can add more embeddings in the future -> this is another category in steps.

Reply:
@jakubczakon check how to make tokenizer without keras